lib/shared/loading.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_spinkit/flutter_spinkit.dart';
+
+class Loading extends StatelessWidget {
```

Comment:
klasy muszą nazywać się od rzeczowników.  teoretycznie Loading to też rzeczownik no ale wiesz.

Reply:
done